lib/vm/src/table.rs

```diff
@@ -88,6 +88,11 @@ impl VMTable {
         unsafe { Self::new_inner(table, style, None) }
     }
 
+    /// Returns the size of the table
+    pub fn get_runtime_size(&self) -> u32 {
```

Comment:
Should we be using `usize` here, or is that a wasm32 thing?

Reply:
No, u32 is correct, this is a wasm32 thing I think.